ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -317,9 +326,48 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
       history.back();
     }
 
+    async initialBillingAccountLoad() {
+      // If user hasn't granted GCP billing scope to workbench, we can not fetch billing account from Google
+      // or fetch user's available billing accounts.
+      // When creating/duplicating workspace, show free tier billing account.
+      // When editing existing workspace, show free tier if that is currently being used or 'User Provided Billing Account'
+      // if it is user's billing account.
+      if (serverConfigStore.get().config.enableBillingUpgrade && !hasBillingScope()) {
+        const freeTierBillingAccount: BillingAccount = {
+          name: 'billingAccounts/' + serverConfigStore.get().config.freeTierBillingAccountId,
+          isFreeTier: true,
+          isOpen: true,
+          displayName: 'Use All of Us initial credits',
```

Comment:
this says `initial` and server says `free`; which is correct?

Reply:
Shimon mentioned free is not correct. And the mock is initial. I am changing our API to initial as well.